tools/build-vendor.js

```diff
@@ -60,6 +60,9 @@ const files = {
   'webext-launch-web-auth-flow': [
     'dist/webext-launch-web-auth-flow.min.js → webext-launch-web-auth-flow.min.js',
   ],
+  'slipjs': [
```

Comment:
40k js library just for drag'n'drop is excessive. It shouldn't be more than a couple of kb.

Also, the entire row should be draggable, not just the icon.

Reply:
I wrote a simple draggable-list using HTML drag-n-drop:
https://eight04.github.io/draggable-list/
WDYT? One difference I can see is that they don't allow you to customize cursor during dragging.

Reply:
Size-wise it's good, but for Stylus we don't need the `iterator` polyfill, so either set the minimum browsers to Chrome55/FF53 or use a non-minified version if it's also small.

I'd use `cursor: move` for the items ![](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor/move.gif), not `pointer`.

When dragging I see constant flickering of the mouse cursor, which alternates between `+` and `blocked circle`. It shouldn't change. If it's caused by gaps between the items then make sure to remove them.

When dragging, the cursor shouldn't be `+` because we don't copy anything. IIRC there's a way to indicate the mode is a move, not copy, in some drag'n'drop event handler.